### PR TITLE
DEP-1789 modify type AppCompatActivity to Activity

### PR DESCRIPTION
**Pull request by marinakinalone**

### CODE REVIEW

**Comment by marinakinalone** [comment]
@jennylarsson94 I tested by adding the modification on the local SDK files in the native test app and it works!

**Comment by jennylarsson94** [comment]
> @jennylarsson94 I tested by adding the modification on the local SDK files in the native test app and it works!

Good! Don't forget to add the changes to the `Changelog.md` file as well and update gradle

File: Changelog.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 - **Gradle**: Updated to version 7.6.4.
 - **Compile SDK API version**: Updated to 33.
 - **Java compile version**: Updated to v17.
-- **Appcompat**: Updated to version 1.6.1.
+- **Appcompat**: Removed.
```
Comment (jennylarsson94) [inline Changelog.md]: This should be moved to a new section - look at the structure of the changelog

File: .vscode/settings.json
Comment (jennylarsson94) [inline .vscode/settings.json:1]: Add a gitignore on this one

**Review by jennylarsson94 — CHANGES_REQUESTED** [review]

**Review by jennylarsson94 — APPROVED** [review]

**Pull request merged**